src/Shared/FileUtilities.cs

```diff
@@ -110,6 +115,17 @@ private static void GetTestExecutionInfo()
         }
 
         /// <summary>
+        /// Returns true if processName appears in the processList
+        /// </summary>
+        /// <param name="processName">Name of the process</param>
+        /// <param name="processList">List of processes to check</param>
+        /// <returns></returns>
+        private static bool IsProcessInList(string processName, string[] processList)
+        {
+            return processList.Any(s => processName?.IndexOf(s, StringComparison.CurrentCultureIgnoreCase) >= 0);
+        }
```

Comment:
process name is not culture specific, I think? shouldn't it be InvariantCultureIgnoreCase